metadata/src/metadata.cpp

```diff
@@ -136,7 +136,7 @@ int write_meta_data_json( creds_fetcher::krb_ticket_info* krb_ticket_info,
  * @return 0 or 1 for successful or failed writes
  */
 int write_meta_data_json( std::list<creds_fetcher::krb_ticket_info*> krb_ticket_info_list,
-                          std::string lease_id, std::string krb_files_dir )
+                           std::string lease_id, std::string krb_files_dir )
```

Comment:
Was this a typo? I think it is now misaligned with the line above.

Reply:
addressed the comment